README.md

```diff
@@ -13,6 +13,32 @@ This is the R package for simulating spread of pests and pathogens. The package
 
 PoPs is a stochastic spread model of pests and pathogens in forest and agricultural landscapes to learn more visit [popsmodel.org](https://popsmodel.org/). The R package provides an easy way for researchers to calibrate, validate, and test what if scenarios of treatment interventions. The model is also available in GRASS GIS you can install and use [r.pops.spread](https://github.com/ncsu-landscape-dynamics/r.pops.spread) to run the model in GRASS GIS.
 
+## Features
+The PoPS Core C++ library and its interfaces: rpops R package and r.pops.spread GRASS GIS module. The release of rpops includes:
+
+* Susceptible-infected (`SI`) and susceptible-exposed-infected (`SEI`) host phases (`model_type`, `latency_period`),
+* Host mortality tracking (`mortality_rate`, `mortality_time_lag`, `mortality`),
+* Host removal and pesticide application treatments (`treatments`, `treatment_date`, `pesticide_duration`),
+* Host resistance based on pesticide application treatments (`pesticide_duration` > 0),
+* Treatments applied only to a ratio of hosts (`treatment_application`),
+* Yearly pest removal based on lethal temperature (`lethal_temperature`, `lethal_month`),
+* Two different dispersal kernels (`natural_dispersal_kernel`, `anthropogenic_dispersal_kernel`),
+* Cauchy, Exponential, Uniform, Power-law, Deterministic neighbor, Hyperbolic-Secant, Gamma, Weibull, Normal, and Logistic radial dispersal kernels use Von Mises distribution,
+* Seasonal spread (`seasonality` in months),
+* Host movement for animals moving from farm to farm or plants via nursery trade (`use_movements`, `movements_file`),
+* Reduced stochasticity options (`generate_stochasticity`, `establishment_stochasticity`, `movement_stochasticity`) and deterministic versions of kernels (`deterministic`) other required parameters if reducing stochasticity are (`dispersal_percentage`, `establishment_probability`),
+* Spread rate measurement in 4 cardinal directions (`west_rate`, `east_rate`, `south_rate`, `north_rate`) when (`use_spreadrates`) is true,
+* Distance to quarantine in 4 cardinal directions (`north_distance_to_quarantine`, `south_distance_to_quarantine`, `east_distance_to_quarantine`, `west_distance_to_quarantine`) when (`use_quarantine`) is true and (`quarantine_areas_file`) is provided,
+* Probability of quarantine escape (`escape_probability`).
+* Overpopulation function (individuals in areas of high population leave the area and disperse longer distances on average) (`use_overpopulation`, `overpopulation_percentage`, `leaving_percentage`, `leaving_scale_coefficient`)
+* Flexible output frequency with n number of days, weeks, months, or years as options (`output_frequency`, `output_frequency_n`)
```

Comment:
Inconsistent punctuation. I think the commas I used before were actually a bad choice for this list. In this case, lower case and no punctuation is probably best, although starting with upper case is fine too. Alternatively, since some are sentences, maybe pretend all are and start with upper case and end with full stop.